packages/temporal/api/mpe_routes.go

```diff
@@ -100,6 +101,50 @@ func createMpeRoomHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(res)
 }
 
+type MpeAddTracksRequestBody struct {
+	WorkflowID string `json:"workflowID" validate:"required,uuid"`
+
+	TracksIDs []string `json:"tracksIDs" validate:"required,dive,required"`
+	UserID    string   `json:"userID" validate:"required"`
+	DeviceID  string   `json:"deviceID" validate:"required"`
+}
+
+func MpeAddTracksHandler(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+
+	var body MpeAddTracksRequestBody
+
+	if err := json.NewDecoder(r.Body).Decode(&body); err != nil {
+		WriteError(w, err)
+		return
+	}
+	if err := validate.Struct(body); err != nil {
+		WriteError(w, err)
+		return
+	}
+
+	signal := shared_mpe.NewAddTracksSignal(shared_mpe.NewAddTracksSignalArgs{
+		TracksIDs: body.TracksIDs,
+		UserID:    body.UserID,
+		DeviceID:  body.DeviceID,
+	})
+	if err := temporal.SignalWorkflow(
+		context.Background(),
+		body.WorkflowID,
+		"",
```

Comment:
should we create something like `shared.NoWorkflowRunIDRelated`

Reply:
That's a good idea, yeah!